example/windows/CheckboxExample.Package/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="CheckboxExample"
+    Publisher="CN=anuverma"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>CheckboxExample</DisplayName>
+    <PublisherDisplayName>anuverma</PublisherDisplayName>
```

Comment:
Example app isn't ever going to be independently published to the store, so would be better to not have a specific individual's publish information here.